tests/unit/einstein/test_ceph.py

```diff
@@ -0,0 +1,272 @@
+import time
+import unittest
+import pytest
+import subprocess
+import ims.common.config as config
+
+config.load()
+
+import rados
+import rbd
+import ims.common.constants as constants
+import ims.einstein.ceph as Ceph
+from ims.database.database import Database
+from ims.einstein.operations import BMI
+import ims.exception.file_system_exceptions as file_system_exceptions
+from ims.common.log import trace
+from pytest import set_trace as bp
+import logging
+
+_cfg = config.get()
+
+PROJECT = _cfg.tests.project
+NETWORK = _cfg.tests.network
+
+EXIST_IMG_NAME = _cfg.tests.exist_img_name
+NEW_SNAP_NAME = _cfg.tests.new_snap_name
+NOT_EXIST_IMG_NAME = _cfg.tests.not_exist_img_name
+NOT_EXIST_SNAP_NAME = _cfg.tests.not_exist_snap_name
+
+HIL_CALL_TIMEOUT = 5  # constants.HIL_CALL_TIMEOUT
+
+POOL = _cfg.fs.pool
+# test image
+CEPH_IMG = "BMI_TEST_CEPH"
+# test image size
+CEPH_IMG_SIZE = 1024
```

Comment:
instead of having comments here, how about we make the names self explanatory. 

Also, CEPH_IMG_SIZE is pretty self explanatory, we don't really need a comment explaining what's it about. Have a comment when you are doing something funky and it's not apparent whats going on.